cellSAM/cellsam_pipeline.py

```diff
@@ -155,7 +188,6 @@ def cellsam_pipeline(
         
         # To prevent creating model for each block
         device = 'cuda' if torch.cuda.is_available() else 'cpu'
-        print("Warning, using standard model. For better performance, use a model trained on your data.")
```

Comment:
If we really want to warn users about this, it should be made a UserWarning so that it can be filtered for the purporses of e.g. testing.

Any opinions @damaggu @ulisrael @rdilip @m274d ? IMO it's fine to remove - at least until the training recipe is uploaded!

Reply:
@rossbar sure, yeah. let's remove it